lib/redcord/vacuum_helper.rb

```diff
@@ -17,7 +17,7 @@ def self.vacuum(model)
       _vacuum_range_index_attribute(model, range_index_attr)
     end
     model.class_variable_get(:@@custom_index_attributes).keys.each do |index_name|
-      puts "Vacuuming custom index: #{index_name}"
+      puts "Vacuuming custom index: #{index_name} for model #{model}"
```

Comment:
```suggestion
      puts "Vacuuming custom index: #{index_name} for model #{model.name}"
```
can we add a model name for other puts statements in this file too?